Simulator/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu
```

Comment:
Can "Dockerfile" also be lowercased to dockerfile or is this the standard?

Reply:
Uppercase d is the standard in this project